typing/btype.ml

```diff
@@ -77,7 +78,7 @@ type change =
   | Ckind of field_kind option ref * field_kind option
   | Ccommu of commutable ref * commutable
   | Cuniv of type_expr option ref * type_expr option
-  | Ctypeset of TypeSet.t ref * TypeSet.t
+(*  | Ctypeset of TypeSet.t ref * TypeSet.t *)
```

Comment:
If this isn't used, why are we leaving it in a comment?

Reply:
@garrigue, I think it would be useful to answer this question (ideally in a comment in the source) or remove the line. Old comments lying around for no clear reason, that maybe we want to keep or maybe not can, turn the source into a minefield for newcomers.

Reply:
Removed the comments. This was dead code since @trefis introduced scopes.